modules/reference/pages/home/my-organization-configuration.adoc

```diff
@@ -1,7 +1,26 @@
 [[ref.webui.overview.org.config]]
 = Organization Configuration
 
-On the menu:Main Menu[Home > My Organization > Configuration] page modify your personal information, such as name, password, and title.
-To modify any of this information, make the changes in the appropriate text fields and click the btn:[Update] button at the bottom.
+The menu:Home[My Organization > Configuration] section allows you to configure your current organization.
 
-image::home_my_organization_configuration.png[scaledwidth=80%]
+[[org-config-options]]
+.Organization Configuration Options
+[cols="1,1,1", options="header"]
+|===
+| Option                            | Description   | Default
+| Enable staging contents | For clients in this organization, allow content staging by default. | Unchecked
+| Enable Errata E-mail Notifications | For users in this organization, send email notifications when errata is available. | Checked
+| Enable Software Crash Reporting | In case of a crash, a log of the crash is saved to file. | Checked
+| Enable Upload of Crash Files | Allow crash log files to be uploaded to {suse}. | Checked
+| Crash File Upload Size Limit | The maximum crash log file size (in MB) that can be uploaded to {suse}. | 2048{nbsp}MB
+| Enable Upload of Detailed SCAP Files | Allow detailed SCAP content files to be uploaded for auditing. | Unchecked
+| SCAP File Upload Size Limit | The maximum SCAP file size (in MB) that can be uploaded. | 2048{nbsp}MB
+| Allow Deletion of SCAP Results | Allow SCAP results to be deleted after the audit is complete. | Checked
+| Allow Deletion After | The number of days after an SCAP audit is complete, that results can be deleted. | 90 days
+|===
+
```

Comment:
I rather vote to use an itemized list instead of single sentence paras for the next "for more info" items.

Reply:
Sure.